src/main/java/fi/aalto/cs/apluscourses/ui/exercise/ExercisesTreeRenderer.java

```diff
@@ -28,7 +30,8 @@ private static Icon statusToIcon(@NotNull ExerciseViewModel.Status exerciseStatu
       case FULL_POINTS:
         return PluginIcons.A_PLUS_FULL_POINTS;
       default:
-        throw new IllegalStateException("Invalid exercise view model status");
+        throw new IllegalStateException(
```

Comment:
perhaps no need to wrap `Exception` messages into props, idk

Reply:
Good point, I'll change this.